README.md

```diff
@@ -539,15 +537,15 @@ Martin Fowler'ın "Code Smell" dediği "Data Class", çoğu yazılımcı tarafı
 
 #### Problem
 
-Bir değişken, parametre, alan, metot veya sınıfın artık kullanılmamasıdır(genellikle artık eskimiş olduğundan).
+Bir değişken, parametre, alan, metot veya sınıfın artık kullanılmamasıdır (genellikle artık eskimiş olduğundan).
 
 #### Sebep
 
 Yazılımın gereksinimleri değiştiğinde veya düzeltmeler yapıldığında, eski kodu temizlemek için hiç kimse zaman harcamak istemez. Bu tür bir kod karmaşık kod bloklarında da bulunabilir.
 
 #### Çözüm
 
-Ölü kodu bulmanın en hızlı yolu iyi bir IDE kullanmaktır. Çözmek ise basit; sil.
+Ölü kodu bulmanın en hızlı yolu iyi bir IDE kullanmaktır. Çözmek ise basit: sil.
```

Comment:
Burada da neden değişiklik olduğunu açıklayabilir misiniz? Noktalı virgül daha mantıklı geldi bana.

Reply:
Türkçe dil kurallarına göre ; (noktalı virgül) ilişkili iki cümleyi birbirinden ayırmak için, : (iki nokta) ise kendisiyle ilgili örnek verilecek cümlenin sonuna konur. Yukarıdaki örnek ikinci bir cümleyle devam etmek yerine örnekle devam ediyor gibi daha çok.